pubsub/averager.go

```diff
@@ -49,10 +49,10 @@ func newAverager(dur time.Duration, nBuckets int) *averager {
 func (a *averager) average() float64 {
```

Comment:
Nit: consider making this `Average` since it's an "external" function? (even though the whole struct isn't). Not sure what typical Go style for this is, but I think it can be useful to be clear about what the expected API of internal objects is even if it's not enforced.

This code could also be moved into the `internal/batcher` package.

Reply:
Removed the whole thing.